user-guide/docs/index.md

```diff
@@ -105,27 +87,27 @@ correct path to where you unzipped the SDK.
 }
 ```
 
+
 ## Installing the Conclave Cloud Java/Kotlin Client SDK
 
-For the beta version of Conclave Cloud, the Java/Kotlin Client SDK is delivered
-in a zip file that contains a file-based maven repository. This can be added as
-a dependency in your Java/Kotlin project by adding the repository then
-describing the dependency.
 
-Download and unzip the SDK to the directory of your choice. Add the repository
-to your `build.gradle.kts`, making sure to provide the correct path to where you
-unzipped the SDK.
+1. Download and unzip [the SDK](https://github.com/R3Conclave/ccl-documentation/releases/download/1.0.0-beta2/conclave-cloud-sdk-java-1.0.0-beta2.zip)
```

Comment:
I think this link is now out of date - shouldn't be `beta2`. 
https://github.com/R3Conclave/ccl-documentation/releases/download/1.0.0-GA/conclave-cloud-sdk-java-1.0.0-GA.zip